cbHasSpace/js/cargobay.has-space.js

```diff
@@ -58,7 +55,7 @@ cargobay.hasSpace = (function(window, undefined) {
         }
     };
 
-    debounce = function(func, wait, immediate){
+    _debounce = function(func, wait, immediate){
```

Comment:
Add a comment giving credit to the original author of this debounce function. This function is copied form https://davidwalsh.name/javascript-debounce-function, in this article he says that he just took it from underscore.js (https://github.com/jashkenas/underscore/blob/master/underscore.js#L852). The function looks a bit different nowadays, but basically does the same thing.